libshaderc/src/shaderc.cc

```diff
@@ -367,31 +368,36 @@ shaderc_spv_module_t shaderc_compile_into_spv(
   return result;
 }
 
-size_t shaderc_module_get_length(const shaderc_spv_module_t module) {
+size_t shaderc_module_get_length(const shaderc_compilation_result_t module) {
```

Comment:
For consistency, all these shaderc_module_* seem like they should be shaderc_result_*